### ISSUE TITLE
Travis test failing: octave package issue

### ISSUE THREAD

**Issue by mathieuboudreau** [qMRLab/qMRLab#351]
Just noticed this here when creating a branch: https://travis-ci.org/qMRLab/qMRLab/jobs/575809658

Getting the following error:

```
$ travis_retry sudo apt-get -y -qq install octave liboctave-dev octave-pkg-dev
Unable to correct problems, you have held broken packages.
The command "sudo apt-get -y -qq install octave liboctave-dev octave-pkg-dev" failed. Retrying, 2 of 3.
```

**Comment by mathieuboudreau** [comment]
Just did that in this commit: https://github.com/qMRLab/qMRLab/commit/02d9e057f41d14ceb63dc590a15879ad4c480c35

Travis log: https://travis-ci.org/qMRLab/qMRLab/builds/575876851?utm_source=github_status&utm_medium=notification

**Comment by notZaki** [comment]
I got the following error:
```
The following packages have unmet dependencies:
 octave-pkg-dev : Depends: debhelper (>= 10) but 9.20160115ubuntu3 is to be installed
```
So octave wants `debhelper10` but xenial only has `debhelper9` [[ref]](https://packages.ubuntu.com/xenial/debhelper)
Luckily, `debhelper10` is in xenial-backports [[ref]](https://packages.ubuntu.com/xenial-backports/debhelper) so I updated the `.travis.yml` to grab the backports version.

The tests still fail, but for a different reason at the very end. Not sure what's going on there.
